tests/driver_bmp180/main.c

```diff
@@ -73,12 +74,12 @@ int main(void)
         /* Get altitude in meters */
         int16_t altitude = bmp180_altitude(&dev, pressure_0);
 
-        printf("Temperature [°C]: %d.%d\n"
+        printf("Temperature [°C]: %i.%d\n"
                "Pressure [hPa]: %lu.%d\n"
                "Pressure at see level [hPa]: %lu.%d\n"
               "Altitude [m]: %i\n"
                "\n+-------------------------------------+\n",
-               (int)(temperature / 10), (int)(temperature % 10),
+               (int)(temperature / 10), abs(temperature % 10),
```

Comment:
What's the need for abs()

Reply:
In the example provided by @benpicco above, without abs, you would get the following result (tested locally):
```
Temperature [°C]: -23.-5
```
But I agree that changing `%d` in `%i` was not necessary...

Reply:
Ah, OK. I wasn't thinking

Reply:
This needs to be fixed in `tests/driver_bmx280/main.c` as well. And maybe there are more